backend/src/util/virtualTransactions.ts

```diff
@@ -33,12 +32,12 @@ const defaultModelFunctions = {
 const virtualLinkTransaction = (
   balance: Decimal,
   amount: Decimal,
-  holdAvailableAmount: Decimal,
+  _holdAvailableAmount: Decimal,
   decay: Decimal,
   createdAt: Date,
   validUntil: Date,
   user: User,
-  previousBalance: Decimal,
+  _previousBalance: Decimal,
```

Comment:
whats the intention or rule to add a prefixed underscore?

Reply:
It is to mark the variable as intentional unused.
But in this case we may remove them.. later in another refactoring round